### DISCUSSION TITLE
Can Brew help with getting a Linux on a Mac display two monitors with a USB 3.0 HDMI adapter that only works with windows PC?

### DISCUSSION THREAD

**Discussion by JPaWebSlay** [Homebrew/discussions#4999] [Everyday usage]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

I have an old 2006-2009 Macbook 6.1 Intel Core Duo2 w a NVdia 9400 video card that I had Linux mint on but no adapter would get me 2 monitors.

**Comment by SMillerDev** [comment]
If nothing on Linux will work, neither will anything on brew.
